rclc/src/rclc/executor.c

```diff
@@ -1075,11 +1630,6 @@ _rclc_execute(rclc_executor_handle_t * handle)
     }   // switch-case
   }
 
-  // corresponding callback of this handle has been called => reset data_available flag here
-  // (see also comment in _rclc_read_input_data() function)
-  if (handle->data_available == true) {
-    handle->data_available = false;
-  }
   return rc;
```

Comment:
why did you remove this code?

Reply:
Change reverted on b812de40f1aa164db58b7a19e60a8ce8ddd95f1a